### ISSUE TITLE
payday 2 ramdomly closing after startup no mods installed refresh install 

### ISSUE THREAD

**Issue by wolfy1992** [Kamikaze94/WolfHUD#242]
Hi kami im having this problem where i stated in the last thread it opens then close well its doing it again but ive no mods nothing its a fresh install done everything i can think off

**Comment by Kamikaze94** [comment]
If you are running without mods (Note that steam doesn't uninstall mods when you uninstall through steam), maybe you should ask in the steam bug reporting for help. (http://steamcommunity.com/app/218620/discussions/14/)
Those guys have a lot more routine with the basic problems the game has and how to fix them.
Maybe this FAQ will help you as well: http://steamcommunity.com/app/218620/discussions/14/412446292758934434/#c412446292758941816

**Comment by ALE854** [comment]
Maybe some help. My friends game launched without any problems today after not been playing the game since last Saturday. My game didn't at first but after verifying my game cache it launches.

**Comment by wolfy1992** [comment]
yea thats all good it launches but when it gets to press any key it closes itself

**Comment by BangL** [comment]
try unplugging your controller

Am 21.04.2017 19:29 schrieb "wolfy1992" <notifications@github.com>:

**Comment by wolfy1992** [comment]
lmao smh

**Comment by ALE854** [comment]
Doesn't it help to verify your game cache?

**Comment by wolfy1992** [comment]
im closing this sick of repeating posts

**Comment by ALE854** [comment]
Sorry wolfy1992 just wanted to help.

**Comment by wolfy1992** [comment]
i know ive done everything possible ive come to a brick wall all my dlcs200 quids worth gone

**Comment by ALE854** [comment]
Found this at the Payday forum: "A warning to BLT users 
Hello heisters ! 

It seems peoples using any kind of BLT mods have been experimenting lately crashes due to « %2 » being typed in both the in-game and lobby chats, a string not well supported by this BLT mod ending up in everyone using this kind of content disconnecting from a heist.

For the stability and safety of your payday 2 experience, it should be wiser to remove BLT mods for the time being. It is also recommended to verify your game since some mods might leave changes to files that can cause crashes even after they are uninstalled, in which case will require you to make a complete reinstallation/verify of the game. For that process, just go in your steam library, right click the payday 2 icon, select properties, then the local files section and finally select the verify button. 

It should be also noticed that pdmod and mod_override contents are safe from these issues and their users won’t suffer any crashes from this « %2 » crash."

I had no problems what so ever during spring break and I played every night then. The crashing began after spring break and my game crashed when I tried to trade a prisoner to let a teammate out of prison. That problem was solved and the game launched every night until last Sunday. From Monday it started to crash but by verifying my game cache it launched again. Tuesday and Wednesday it didn't launch. Thursday it launched normally again and yesterday I had to verify my game cache again to get it launching. Strangest thing is that there haven't been any upgrades of the game during last week so why is it so unstable when it worked perfectly during spring break and all of the downloads beeing released?
